File: array/largestElement.java
public class largestElement{
    public static void main(String [] args ){
        int [] arr = {12,34,65,1,2,3,456,122,87787,7,87,98778,7,8,78,98};
        int max = arr[0] ;

        for(int i = 1 ; i < arr.length ; i++){
            if(arr[i] > max ){
                max = arr[i];
            }
        }

        System.out.println("The max element is : "+ max );
    }
}

File: array/largestElement.cpp
#include<iostream>
using namespace std ;
int main(){
    int arr[] = {1,2,3,4,5,5,6,6,7,7,8};
    int size = sizeof(arr)/ sizeof(arr[0]);
    int max = arr[0] ;

    for(int i = 1 ; i < size ; i++){
        if(arr[i] > max ){
            max = arr[i] ;
  
        }
    }
   
   cout<< "the highest value is : "<< max ;

}